src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,257 @@
+#Contract Entities
+####AuditableEntity
+#####What is it?
+`AuditableEntity` is the abstract base type for all OpenHDS entities. The class provides fields useful for ‘auditing’ or keeping track of meta information pertaining to the concrete type like when it was created and last modified.
+######Required Fields:
++ String uuid 
++ User insertBy 
++ Date insertDate 
++ User lastModifiedBy 
++ Date lastModifiedDate 
+
+___
+####AuditableCollectedEntity
+(child of `AuditableEntity`)
+#####What is it?
+`AuditableCollectedEntity` is the abstract base type for all OpenHDS entities that are collected by a `FieldWorker`.
+######Dependencies
++ FieldWorker
+
+######Required Fields:
++ FieldWorker collectedBy 
++ Date collectionDateTime
+
+___
+####AuditableExtIdEntity
+(child of `AuditableCollectedEntity`) 
+#####What is it?
+`AuditableExtIdEntity` is an entity that is identifiable by an id external to the OpenHDS, i.e. the functionality of the OpenHDS is not dependent on it. 
+######Required Fields:
++ String extId 
+
+#Census Entities
+####LocationHierarchy
+(child of `AuditableExtIdEntity`)
+#####What is it?
+`LocationHierarchy` is a node in the tree that represents the study area. For example, at the top of a tree would be the The United States and it would have a child node for each of the 50 states. Each state node could have a child node for its cities and each city could have a child node for its districts and so on. `LocationHierarchy` extends `AuditableExtIdEntity` meaning it is identified by uuid as well as extId. 
+#####Dependencies
++ FieldWorker
++ LocationHierarchy
++ LocationHierarchyLevel
+
+######Required Fields:
++ String name
++ String extId
++ LocationHierarchy parent
++ LocationHierarchyLevel level
+
+___
+####LocationHierarchyLevel
+(child of `AuditableEntity`)
+#####What is it?
+The levels of the `LocationHierarchy` tree are defined separately as `LocationHierarchyLevel`s and referenced by each instance of `LocationHierarchy`. From the previous example, the list of `LocationHierarchyLevel`s would be Country, State, City, District.
+######Required Fields:
++ int keyIdentifier
++ String name
+
+___
+####Location
+(child of `AuditableExtIdEntity`)
+#####What is it?
+`Location` is the leaf node on the `LocationHierarchy` tree, it is the smallest geographical unit within a study area. From the previous example for `LocationHierarchy`, each building within a district could be modeled with `Location`. `Individual`s are residents at `Locations`.
+######Dependencies
++ FieldWorker
++ LocationHierarchy
++ Domain Constraints
+
+######Required Fields:
++ String name
++ String extId
++ LocationHierarchy parent
+
+___
+####Individual
+(child of `AuditableExtIdEntity`)
+#####What is it?
+An `Individual` (person) within the study area that resides at a `Location`, is a member of a `SocialGroup`, and has a relationship to the head of a household.

```

Comment:
I think it's confusing to mention the head of the household here.  The reader doesn't know what an OpenHDS household consists of yet.